harness.sh

```diff
@@ -31,7 +31,7 @@ LAST_N_PERIOD=5m
 GUI_DIR="${HARNESS_ROOT}/gui"
 
 # Using postgres requires the DB specified below to exist and contain no data.
```

Comment:
This comment is no longer true. Previously the database had to be empty, but now it does not because it will be automatically purged.